### DISCUSSION TITLE
Heizen (Auto, Kühle usw)

### DISCUSSION THREAD

**Discussion by lutz1299** [brutella/hkknx-public#177] [Ideas]
Es wäre gut, wenn man auswählen könnte das eine Heizung nur Heizen und nicht kühle kann. In der Thinka ist es einzustellen.

**Comment by mbrockeu** [comment]
Aktuell wird das doch eh nicht durchgereicht oder?

**Comment by brutella** [comment]
Diese Einstellungen für Thermostate sind seit Version [2.7.0](https://github.com/brutella/hkknx-public/releases/tag/2.7.0) verfügbar.

**Comment by sirmoon137** [comment]
@brutella  Hallo, was mach ich falsch ich hatte auch thinka da funktionierte das stellen des Stellwerts in der Heizung ohne Probleme ich bekomm es leider mit hkknx nicht hin.

hier ein screenshot wie es in thinka aus gesehen hat ich weiss nur nicht wie ich dies in hkknx nachbilden soll. 
<img width="1728" alt="thinka_Heizung_Wohnzimmer" src="https://github.com/user-attachments/assets/f72cde4a-7f49-446f-9405-b3b4b6cf8179">

**Reply by brutella** [reply]
Ich würde das in hkknx wie folgt konfigurieren.

- Ist-Temperatur
    - Lesen: 6/2/10
    - Initialisieren: 6/2/10
- Soll-Temperatur: 
    - Schreiben: ? ← fehlt?
    - Lesen: 6/4/21
    - Initialisieren: 6/4/21
- Status: Heizen (DPT-1)
    - Schreiben: ? ← fehlt?
    - Lesen: 6/4/24
    - Initialisieren: 6/4/24
- Status: Aus (DPT-1-3)
    - Schreiben: ? ← fehlt?
    - Lesen: 6/4/24
    - Initialisieren: 6/4/24
- Modus Komfort/Standby/Economy/Forst-/Hitzeschutz
    - Schreiben: 6/4/27
    - Lesen: 6/4/27
    - Initialisieren: 6/4/27

**Erläuterung**
- Die Gruppenadresse zum Schreiben der absoluten Soll-Temperatur (siehe Soll-Temperatur Schreiben) fehlt noch. Hast du dazu eine Gruppenadresse?
- Da beim Status Heizen und Aus nur eine Lese-Gruppenadresse zur Verfügung steht, kann man zwar in der Apple Home-App zwischen Heizen und Aus wählen, es passiert dann in KNX aber nichts. Wenn du gerne den Status selbst setzen möchtest, benötigst du dazu noch eine Schreiben-Gruppenadresse.

**Comment by sirmoon137** [comment]
Also das setzen der Soll Temperatur so weit ich das verstehe ist in Thinka über den "setpoint shift" gemacht

**Comment by sirmoon137** [comment]
In ETS schaut es für meinen B.IQ aktuell so aus. 
<img width="921" alt="Bildschirmfoto 2024-10-23 um 13 21 49" src="https://github.com/user-attachments/assets/43b9e17c-4ad0-441e-963d-fd98024943df">

**Reply by brutella** [reply]
Kannst du mal `6/4/23` für die Soll-Temperatur testen?

**Reply by sirmoon137** [reply]
cool das scheint, zumindest teilweise, zu funktionieren. Also RTR übernimmt jetzt den wert den ich im HomeKit setze, jedoch springt der wert den ich am iphone setze nach einer sekunde um 0,5 grad nach oben, sprich wenn ich 22,0 setze hüpft er nach einer secunde auf 22,5

**Comment by sirmoon137** [comment]
@brutella setpoint shift kann ich in hkknx nicht verwenden, hast du sonst eine idee?
